src/client/pages/OfferNew/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Tabs } from './Tabs'
+const mockInsurances = [
+  {
+    id: '1',
+    name: 'Hemförsäkring',
+    selected: true,
+  },
+  {
+    id: '2',
+    name: 'Olycksfallsförsäkring',
+    selected: false,
+  },
+  {
+    id: '3',
+    name: 'Inbo',
+    selected: false,
+  },
+]
+
+export default {
+  title: 'Offer/Tabs',
+  component: Tabs,
+  parameters: {
+    backgrounds: { default: 'gray100' },
+  },
+}
+
+export const Default = () => {
+  const [insurances, setInsurances] = React.useState(mockInsurances)
+
+  const handleOnChange = (id: string) =>
+    setInsurances((prev) =>
+      prev.map((item) => ({ ...item, selected: item.id === id })),
+    )
+
+  return <Tabs insurances={insurances} onChange={handleOnChange} />
+}
```

Comment:
Perhaps show here also how we can use this setup to show different content based on selected tab?